oidc_provider/admin.py

```diff
@@ -52,7 +52,7 @@ class ClientAdmin(admin.ModelAdmin):
     fieldsets = [
         [_(u''), {
             'fields': (
-                'name', 'owner', 'client_type', 'response_type', '_redirect_uris', 'jwt_alg',
+                'name', 'owner', 'client_type', 'response_types', '_redirect_uris', 'jwt_alg',
```

Comment:
How its presented to the user on admin? via select? raw id?

Reply:
It's using the multiselect mechanism that is built in to django for many to many relationships:
![image](https://user-images.githubusercontent.com/1438495/44235976-10cc4900-a171-11e8-82e8-7199193f5a1d.png)

On the admin list page it then show the descriptions similar to before:
![image](https://user-images.githubusercontent.com/1438495/44236114-7ddfde80-a171-11e8-8508-1e68c1f4d78e.png)

Good question. I'll also update the PR description with this.